### DISCUSSION TITLE
Sliders with non 1x velocity have incorrect size on timeline while placing

### DISCUSSION THREAD

**Discussion by Feodor0090** [ppy/osu#20127] [Q&A] [answered]
https://user-images.githubusercontent.com/53872073/188335147-327be7cb-fc67-4130-95a6-82af427b0efc.mp4

Their duration seems to be calculated as they were 1x, but after placement finish it got recalculated and the sliders are getting correct sizes on timeline.

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/20133
